devel/kokkos/Portfile

```diff
@@ -29,8 +45,21 @@ configure.args-append       -DBUILD_SHARED_LIBS=ON \
                             -DKokkos_ENABLE_SERIAL=ON \
                             -DKokkos_ENABLE_HWLOC=ON
 
+if {[string match *gcc* ${configure.compiler}]} {
+    configure.ldflags-append \
+                            -latomic
+}
+
+if {[string match *clang* ${configure.compiler}]} {
+    configure.ldflags-append \
+                            -L${prefix}/lib/libomp \
+                            -lomp
+}
```

Comment:
an this not be done automagically by setting "compiler.openmp_version" instead?

Reply:
I will check. I have seen other ports adding this manually, see. `libfaiss`.

Reply:
Perhaps not needed, let us see if everything builds. Rebased now.

Reply:
@reneeotten Well, no, at least for `kokkos-devel` without explicit linking it fails:
```
Undefined symbols for architecture x86_64:
    "___kmpc_for_static_fini", referenced from:
        _.omp_outlined. in Kokkos_HostSpace_deepcopy.cpp.o
        _.omp_outlined..7 in Kokkos_HostSpace_deepcopy.cpp.o
        _.omp_outlined..8 in Kokkos_HostSpace_deepcopy.cpp.o
    "___kmpc_for_static_init_8u", referenced from:
        _.omp_outlined. in Kokkos_HostSpace_deepcopy.cpp.o
        _.omp_outlined..7 in Kokkos_HostSpace_deepcopy.cpp.o
        _.omp_outlined..8 in Kokkos_HostSpace_deepcopy.cpp.o
```